linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_space_left_percentage/tests/no_percent_sign.fail.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_stig
```

Comment:
According to https://complianceascode.readthedocs.io/en/latest/tests/README.html#scenarios-format you shouldn't use `profiles` in test scenarios headers unless you're making a regression test for a bug in a profile. You probably want to use `variables` instead and specify the value here.